code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -142,6 +142,46 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "glass of whiskey"
 	glass_desc = "The silky, smokey whiskey goodness inside the glass makes the drink look very classy."
 	shot_glass_icon_state = "shotglassbrown"
+	var/heal_det_brute_amt = 0.5
+	var/heal_det_burn_amt = 0.5
+
+/datum/reagent/consumable/ethanol/whiskey/on_mob_life(mob/living/carbon/M)
+	if(M.mind && M.mind.assigned_role == "Detective")
+		M.heal_bodypart_damage(heal_det_brute_amt,heal_det_burn_amt) //not a painkiller but it'll have to do
+		. = 1
+	return ..() || .
+
+/datum/reagent/consumable/ethanol/whiskey/kong
+	name = "Kong"
+	description = "Makes You Go Ape!�"
+	color = "#332100" // rgb: 51, 33, 0
+	quality = DRINK_BAD //you don't drink it for the taste
+	addiction_threshold = 30 //min pleasure
+	taste_description = "the vice grip of a giant ape"
+	glass_name = "glass of Kong"
+	glass_desc = "When you have a hole in your second favorite drinking arm, only one brand will do."
+	heal_det_brute_amt = 1
+	heal_det_burn_amt = 1
+
+/datum/reagent/consumable/ethanol/whiskey/candycorn
+	name = "candy corn infused whiskey"
+	description = "Just like they drank in the speakeasies, probably."
+	color = "#ccb800" // rgb: 204, 184, 0
+	quality = DRINK_BAD //it can't taste good
+	taste_description = "pancake syrup"
+	glass_name = "glass of candy corn infused whiskey"
+	glass_desc = "A glass of putrid looking alcohol that makes you imagine."
+	heal_det_brute_amt = 2
+	heal_det_burn_amt = 2
+	var/incr_det_hal_prob = 20
+	var/incr_det_hal_amt = 4
+
+/datum/reagent/consumable/ethanol/whiskey/candycorn/on_mob_life(mob/living/carbon/M)
```

Comment:
just make it make you hallucinate as much as mindbreaker toxin does

just giving it a CHANCE to increment your hallucination value every 2 seconds is a very minor downside